lms/static/styles/frontend_apps.scss

```diff
@@ -14,27 +8,28 @@
 @use 'components/heading';
 @use 'components/label';
 
-// Generic React components.
-// Note: Some of these are also used by server-rendered HTML templates, not
-// just by React code.
+// Components rendered both by Preact and other HTML (eg. server-rendered
```

Comment:
This might be a little clearer to read if it started with `Styling for components rendered both by...` (Trivial nit)